docs/get-started/apps-integrations.md

```diff
@@ -1,19 +1,19 @@
 ---
 id: apps-integrations
-title: Installing Apps and Integrations
-sidebar_label: Installing Apps
+title: Installing apps and Integrations
```

Comment:
Since this is the formal doc title, it should be capitalized. Sidebar and other headers are ok.
```suggestion
title: Installing Apps and Integrations
```